stable/inbucket/templates/deployment.yaml

```diff
@@ -26,29 +26,29 @@ spec:
         - name: {{ template "inbucket.name" . }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: "{{ .Values.image.pullPolicy }}"
-          env:
-          {{- range $key, $value := .Values.env }}
-            - name: "{{ $key }}"
-              value: "{{ $value }}"
-          {{- end }}
+          envFrom:
+          - configMapRef:
+              name: {{ template "inbucket.name" . }}-configmap
           ports:
             - name: http
-              containerPort: 10080
+              containerPort: {{ .Values.service.port.http }}
```

Comment:
you can configure with env vars which port the http/smtp/pop will run and if you define another port here it will break and not work.

Do you have a suggestion? how can avoid that, but make the user define in which port they want that.

thanks again for your review

Reply:
if the http/smtp/pop port is a separate variable it could be used in env as well as in the deployment.yaml for the container port.

On the other side the *service* port (in https://github.com/helm/charts/blob/master/stable/inbucket/templates/service.yaml) does not have to be the same port, and it can use a named port instead of the container port so that it is not dependent on the actual value and be set independently, see: https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service

Reply:
@davidkarlsen thanks for the comments:
the ports are env variables to set the app but it comes in this format:
```
INBUCKET_SMTP_ADDR: "0.0.0.0:2500"
INBUCKET_POP3_ADDR: "0.0.0.0:1100"
INBUCKET_WEB_ADDR: "0.0.0.0:9000"
```
but for this app, imho, tying the service port to the container port makes sense because the user set which ports he/she wants to run the app, ie, set the http to 9000 and wants to access the service with 9000 and not 10090 or other.

I'm sorry, but this part is not very clear to me.

Reply:
done